src/sdk/pynni/nni/nas/benchmark/nasbench101/model.py

```diff
@@ -0,0 +1,101 @@
+import os
+
+from peewee import CharField, FloatField, ForeignKeyField, IntegerField, Model
+from playhouse.sqlite_ext import JSONField, SqliteExtDatabase
+
+from nni.nas.benchmark.constants import DATABASE_DIR
+
+db = SqliteExtDatabase(os.path.join(DATABASE_DIR, 'nasbench101.db'), autoconnect=True)
+
+
+class Nb101RunConfig(Model):
+    """
+    Run config for NAS-Bench-101.
+
+    Attributes
+    ----------
+    arch : dict
+        A dict with keys ``op1``, ``op2``, ... and ``input1``, ``input2``, ... Vertices are
+        enumerate from 0. Since node 0 is input node, it is skipped in this dict. Each ``op``
+        is one of :const:`nni.nas.benchmark.nasbench101.CONV3X3_BN_RELU`,
+        :const:`nni.nas.benchmark.nasbench101.CONV1X1_BN_RELU`, and :const:`nni.nas.benchmark.nasbench101.MAXPOOL3X3`.
+        Each ``input`` is a list of previous nodes. For example ``input5`` can be ``[0, 1, 3]``.
```

Comment:
could you give a complete example of `arch`? it is fine to have only several nodes